introduction/index.rst

```diff
@@ -4,192 +4,155 @@ Introduction to the SOF Project
 ###############################
 
 |SOF| (SOF) is an open source audio Data Signal Processing (DSP) firmware
-infrastructure and SDK that offers a single code base for all Intel
-hardware platforms. SOF provides infrastructure, real-time control pieces, and
-audio drivers as a community project.
+infrastructure and SDK. SOF provides infrastructure, real-time control pieces,
+and audio drivers as a community project. The project is governed by the |SOF|
+|TSC| (TSC) who are prominent and active developers from the community.
+SOF is developed in public and hosted on the github platform.
 
 The firmware and SDK are intended for developers who are interested in
 audio or signal processing on modern DSPs or who are interested in
-microkernels that run on small but powerful processors.
+microkernels that run on small but powerful processors. SOF provides a
+framework where audio developers can create, test and tune :-
```

Comment:
Line 15: Do no follow colon ":" with hyphen "-"

Reply:
he's been doing this for as long as I can remember, must be some secret encoding

Reply:
long time guilty of this :)